### ISSUE TITLE
Dimension errot in actor-critic.ipynb

### ISSUE THREAD

**Issue by jiang-yuan** [higgsfield/RL-Adventure-2#24]
log_probs = torch.cat(log_probs)
    returns = torch.cat(returns).detach()
    values = torch.cat(values)

    advantage = returns - values

    actor_loss = -(log_probs * advantage.detach()).mean()

the size of log_probs is [80] and the size of advantage is [80,1]
I find that the size of  log_probs * advantage.detach() is [80,80] when I debugging. It is incorrect since the right dimension of log_probs * advantage should be [80] or [80,1]
